cmd/ignite-spawn/ignite-spawn.go

```diff
@@ -80,22 +76,14 @@ func StartVM(vm *api.VM) error {
 
 func serveMetrics(metricsSocket string) {
 	go func() {
-		// create a new registry and http.Server. don't register custom metrics to the registry quite yet
+		// Create a new registry and http.Server. Don't register custom metrics to the registry quite yet.
 		_, server := prometheus.New()
 		if err := prometheus.ServeOnSocket(server, metricsSocket); err != nil {
 			log.Errorf("prometheus server was stopped with error: %v", err)
 		}
 	}()
 }
 
-func patchRunning(vm *api.VM, ipAddrs []net.IP) error {
-	return patchVM(vm, func(patchVM *api.VM) error {
-		patchVM.Status.Running = true
-		patchVM.Status.IPAddresses = ipAddrs
-		return nil
-	})
-}
-
 func patchStopped(vm *api.VM) error {
 	return patchVM(vm, func(patchVM *api.VM) error {
```

Comment:
Now that the patch is only done in one place, and all the desired values are predictable, you can apply a static patch that you've defined in a string

Reply:
Done in 872516a.